src/Framework/GeneratedFileUsedEventArgs.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Build.Framework
+{
+    /// <summary>
+    /// Arguments for the generated file used event
+    /// </summary>
+    [Serializable]
```

Comment:
@JanKrivanek do we still need to mark eventargs classes as serializable?

Reply:
Removed the attribute

Reply:
Looks like it needs the serializable attribute. Here is the error message 
```
(NETCORE_ENGINEERING_TELEMETRY=Restore) Event type "GeneratedFileUsedEventArgs" was expected to be serializable using the .NET serializer. The event was not serializable and has been ignored.
```
Reverting the changes

Reply:
Thanks @JanKrivanek !
Done. And on top of that, as we discussed, I skip `GeneratedFileEventArgs` `TaskFactoryLoggingHost.IsEventSerializable` in order to remove serializable attribute